test/e2e/policies.test-suite/balloons/n4c16/test13-cacheclusters/balloons-2cpu-cacheclusters.cfg

```diff
@@ -0,0 +1,39 @@
+config:
+  allocatorTopologyBalancing: true
+  reservedResources:
+    cpu: 750m
+  pinCPU: true
+  pinMemory: true
+  balloonTypes:
+  - name: l2burst
+    minCPUs: 1
+    maxCPUs: 4
+    shareIdleCPUsInSame: l2cache
+    preferSpreadingPods: true # no "same pod" affinity
+    preferNewBalloons: true # => every container to its own balloon instance
+  - name: l2pack
+    allocatorTopologyBalancing: false # create instances to the same L2 group
+    shareIdleCPUsInSame: l2cache
+    maxCPUs: 1
+  log:
+    debug:
+      - resource-manager
+      - cache
+      - policy
+      - sysfs
+    source: true
+    klog:
+      skip_headers: true
+extraEnv:
+  OVERRIDE_SYS_CACHES: '''[
+  {"cpusets": ["0-7", "8-15"], "level": 3, "size": "128M"},
+
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 2, "size": "4M", "kind": "u"},
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 2, "size": "4M", "kind": "u"},
+
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 1, "size": "256k", "kind": "d"},
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 1, "size": "128k", "kind": "i"},
+
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 1, "size": "256k", "kind": "d"},
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 1, "size": "128k", "kind": "i"}
+  ]'''
```

Comment:
Is this cache configuration override interesting/useful ? Doesn't it provide the same grouping as the socket and the physical cores ?